udemy_exercises/cell_phone/Main.java

```diff
@@ -0,0 +1,89 @@
+import java.util.Scanner;
+
+public class Main {
+
+    private static Scanner scanner = new Scanner(System.in);
+    private static Cellphone myPhone = new Cellphone(1234567890);
+
+    public static void main(String[] args) {
+        boolean active = true;
+        Options();
+        while (active) {
```

Comment:
Consider the two variables 'active' and 'action'.  The 'action' variable determines what to execute, while 'active' indicates when to exit the loop.  How might the code be refactored to require only one of these two variables?